.github/workflows/ci.yaml

```diff
@@ -107,6 +107,71 @@ jobs:
             # cache node modules using the same key as restore.
           key: ${{ steps.node_modules-cache-restore.outputs.cache-primary-key }}
 
+  E2E-Test:
+    timeout-minutes: 60
```

Comment:
This seems like a super long time out... Something like 5 to 10 mins seems more reasonable?

Reply:
10 minutes seems a bit low to me. Maybe 15 minutes would be good?

Reply:
the default is 360 mins? Wow.... However long we set this if something goes wrong in the job and it decides to hang. it will wait that long before killing it, which means our CI will also hang for that amount of time. 10 to 15 ish mins sounds good I suppose